src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -159,7 +159,7 @@ env | grep -E "^PAI|PATH|PREFIX|JAVA|HADOOP|NVIDIA|CUDA" > /etc/environment
 function run_user_command()
 {
   printf "%s %s\n\n" "[INFO]" "USER COMMAND START"
-  {{{ taskData.command }}} || exit $?
+  eval {{ taskData.command }} || exit $?
```

Comment:
environment variables won't work if you escape '$'

Reply:
```
bash-3.2$ eval echo\ \$HOME
/Users/gerhut
```